tpot/base.py

```diff
@@ -1230,6 +1257,10 @@ def _random_mutation_operator(self, individual, allow_shrink=True):
             ind = self._toolbox.clone(individual)
             offspring, = mutator(ind)
             if str(offspring) not in self.evaluated_individuals_:
+                # Update statistics
```

Comment:
More descriptive comment, please.

Reply:
Ping on this change request, @bartdp1.

Reply:
Missed it, Will do tomorrow

Reply:
It's already there, in one of the last commits :)